docs/core_data_structure.md

```diff
@@ -1,6 +1,186 @@
 (data-structure)=
 # Core data structures
 
-The whiteboard image below captures the planned core data structure of the `Survey` object that is central to Echopop. Most of the structure is in place but we will fully implement them very soon (tracked in issue [#169](https://github.com/OSOceanAcoustics/echopop/issues/169)).
+Input data, intermediate variables and calculations, and results are stored within several attributes associated with the `Survey`-class object. Information on how these attributes are used in the general `echopop` workflow can be found in greater detail [within the example notebook](echopop-example-workflow). 
 
-![ text ](images/core_data_structure.jpg)
+## Class attributes
+
+There are currently **five** unique class attributes that store different types of information and data. 
+
+* `Survey.meta`: Necessary information such as the date the object was created and general data workflow/provenance would be collected.
+* `Survey.config`: Background configuration settings. 
+* `Survey.input`: Imported acoustic, biological, kriging, and stratification [data detailed elsewhere](input-files).
+* `Survey.analysis`: Intermediate data products, variables, and other calculations that may be both relevant to users (e.g. number proportions) or are required for computing the final results. 
+* `Survey.results`: Overall results for each analysis.
+
+Each of these attributes are organized using nested dictionaries that may contain an uneven number of levels. For instance, calculated number and weight proportions computed via the transect analysis are nested via the following structure:
+* Survey object
+  * `analysis` attribute
+    * `transect` key
+      * `biology` sub-key
+        * `proportions` sub-key
+          * `number` sub-key
+            * Various number proportion `DataFrame`'s
+          * `weight` sub-key
+            * Various weight proportion `DataFrame`'s
```

Comment:
I would suggest removing the attribute and sub-key specifiers to avoid confusion. These are all dictionary keys all the down to each of the "leaves", and as you know, key-value has very specific meaning in python.